portality/templates/application_form/editorial_form_body.html

```diff
@@ -1,3 +1,10 @@
+{% if obj.application_status == "update_request" and (not current_journal or not current_journal.is_in_doaj()) %}
```

Comment:
I'm not sure whether `current_journal` is passed into the template, should it be `obj.current_journal`?

Reply:
Also, I believe we have access to `constants` here so we should use that instead of `"update_request"`

Reply:
We **do** have access to `current_journal` in this context

Reply:
`obj.application_status` isn't going to be reliable here, as the status of an item changes after it goes through triage.  There are 2 ways to deal with this:

1. Ask the object if it has the **property** `current_journal`
2. Merge this code with the other PR for distinguishing URs from new applications, and then use `obj.admin.application_type=="update_reqeust"` (but using `constants`)

The first object is probably sufficient, because `current_journal` only gets removed when the item is accepted or rejected, though (2) would be a better overall longer term solution

Reply:
So, to be clear on (1) we would do

```
{% if obj.admin.current_journal is not None and (not current_journal or not current_journal.is_in_doaj()) %}
```

Reply:
I'm going to implement the first solution now to not mix up PRs before they are ready. I guess we can change it once everything is merged and working.